third_party/move/move-prover/tests/sources/functional/schema_name.exp

```diff
@@ -1,6 +1,9 @@
 Move prover returns: exiting with model building errors
-error: Variable `c` conflicts with a specification variable in the schema TestSchemaName::TestNameConflict
-  ┌─ tests/sources/functional/schema_name.move:9:16
-  │
-9 │             x: c + 3
-  │                ^^^^^
+error: A specification variable in the schema TestSchemaName::TestNameConflict conflicts with
```

Comment:
Looked like this sentence ended abruptly. It is not immediately clear that I am supposed to continue reading to the last line.

Reply:
I tried to fix this code, but i'm not sure how to make this file regenerate.

Reply:
Someone with prover experience could help? @rahxephon89